docs/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md

```diff
@@ -12,209 +13,416 @@ AKS main features are:
+**Figure 4-37**. Selecting ASP.NET CORE 3.1 and Web API project type
 
-If you have any previous version of .NET Core, you can download and install the 2.2 version from <https://dotnet.microsoft.com/download>.
+Notice we're not enabling Docker support now, because we'll do it in a moment, just to show it can be done after project creation.
 
-You can add Docker support when creating the project or afterwards, so you can "Dockerize" your project at any time. To add Docker support after project creation, right-click on the project node in Solution Explorer and select **Add** > **Docker support** on the context menu.
+If you have any previous version of .NET Core, you can download and install the 3.1 version from <https://dotnet.microsoft.com/download>.
 
-![Context menu option to add Docker support to an existing project: Right click (on the project) > Add > Docker Support.](media/add-docker-support-to-project.png)
+To show you can "Dockerize" your project at any time. We'll add Docker support now, so right-click on the project node in Solution Explorer and select **Add** > **Docker support** on the context menu.
 
-**Figure 4-38**. Adding Docker support to existing project
+![Context menu option to add Docker support to an existing project: Right click (on the project) > Add > Docker Support.](media/build-aspnet-core-applications-linux-containers-aks-kubernetes/add-docker-support-to-project.png)
 
-To complete adding Docker support, you can choose Windows or Linux. In this case, select **Linux**, because AKS doesn't support Windows Containers (as of late 2018).
+**Figure 4-38**. Adding Docker support to an existing project
 
-![Option dialog to select Target OS for Dockerfile.](media/select-linux-docker-support.png)
+To complete adding Docker support, you can choose Windows or Linux. In this case, select **Linux**. AKS support for Windows container is currently in preview (as of early 2020).
+
+![Option dialog to select Target OS for Dockerfile.](media/build-aspnet-core-applications-linux-containers-aks-kubernetes/select-linux-docker-support.png)
 
 **Figure 4-39**. Selecting Linux containers.
 
-With these simple steps, you have your ASP.NET Core 2.2 application running on a Linux container.
+With these simple steps, you have your ASP.NET Core 3.1 application running on a Linux container.
+
+In a similar way, we add a very simple WebApp application to consume the WebApi endpoint, although we don't get into the details here.
+
+After adding the WebApp project we add orchestrator support as shown next, in image 4-40.
+
+![Adding orchestrator support to WebApi project](media/build-aspnet-core-applications-linux-containers-aks-kubernetes/add-orchestrator-support.png)
+
+**Figure 4-40**. Adding orchestrator support to WebApi project.
+
+When we choose the `Docker Compose` option, which is fine for local development, Visual Studio adds the docker-compose project, with the docker-compose files as shown in image 4-41.
```

Comment:
Avoid "We". The doc started as "You". Rephrase this sentence and the other paragraphs that is referring to "we".
See the tone guide: https://docs.microsoft.com/en-us/contribute/dotnet/dotnet-voice-tone#appropriate-style-1

Reply:
Updated.